apps/scripts/src/limit-redis.js

```diff
@@ -62,6 +113,8 @@ const limit = async () => {
   }
 };
 
+const limit = Promise.all.bind(Promise, [await limitNormal(), await limitHistorical()]);
```

Comment:
```suggestion
const limit = Promise.all.bind(Promise, [limitNormal(), limitHistorical()]);
```

Shouldn't it be this?